Makefile

```diff
@@ -53,7 +53,7 @@ OV_SOURCE_BRANCH ?= master
 DLDT_PACKAGE_URL ?=
 OV_USE_BINARY ?= 1
 YUM_OV_PACKAGE ?= intel-openvino-runtime-centos7
-
+APT_OV_PACKAGE ?= intel-openvino-runtime-ubuntu20-2021.3.394
```

Comment:
2021.3?

Reply:
This is the latest we got. Will update in future.